cli/pcluster/utils.py

```diff
@@ -256,43 +256,6 @@ def _get_json_from_s3(region, file_name):
     return json.loads(file_contents)
 
 
-def get_supported_features(region, feature):
```

Comment:
Note: in the test_validators module there is a mock of this function, to be removed:
`mocker.patch("pcluster.config.validators.get_supported_features ...`

Reply:
done